src/screens/FeedScreen/ContentLink.js

```diff
@@ -150,11 +163,16 @@ const styles = StyleSheet.create({
     letterSpacing: 0.1,
     color: COLORS.white
   },
-  topicStyle: {
-    position: 'relative',
-    marginLeft: 0
-  },
+  topicStyle: (hasImage) => ({
+    position: hasImage ? 'absolute' : 'relative',
+    marginLeft: hasImage ? 12 : 0,
+    bottom: hasImage ? 4 : 0
+  }),
   topicItemStyle: {
     marginBottom: 0
-  }
+  },
+  containerImage: (isBlurred) => ({
+    flex: 1,
+    height: isBlurred ? dimen.normalizeDimen(422) : 100
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These changes in the style object are related to the new image handling logic. The `topicStyle` function now takes a parameter to determine whether an image exists or not, and adjusts the style accordingly. A new `containerImage` style function has been added to handle the styling of the image container. These changes seem appropriate given the new functionality.
```diff
+ topicStyle: (hasImage) => ({
+   position: hasImage ? 'absolute' : 'relative',
+   marginLeft: hasImage ? 12 : 0,
+   bottom: hasImage ? 4 : 0
+ }),
+ containerImage: (isBlurred) => ({
+   flex: 1,
+   height: isBlurred ? dimen.normalizeDimen(422) : 100
+ })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->